dp_mobility_report/model/preprocessing.py

```diff
@@ -116,19 +138,37 @@ def assign_points_to_tessellation(
     )
 
     # Spatial join points to polygons
-    df = gpd.sjoin(
+    df = gpd.sjoin_nearest(
```

Comment:
sjoin_nearest does not provide the correct functionality. Points outside the given tessellation should not be matche to a tile